inference-engine/src/inference_engine/include/ie/ie_plugin_config.hpp

```diff
@@ -283,6 +283,16 @@ DECLARE_CONFIG_VALUE(CPU_THROUGHPUT_AUTO);
  */
 DECLARE_CONFIG_KEY(PERF_COUNT);
 
+/**
+ * @brief The name for setting work mode in MULTI device plugin option.
+ *
+ * It is passed to Core::SetConfig(), this option should be used with value only:
+ * PluginConfigParams::AUTO or PluginConfigParams::MULTI
+ */
+DECLARE_CONFIG_KEY(WORK_MODE);
+DECLARE_CONFIG_VALUE(AUTO);   // fixme: correct me with more meaningful name, thanks!
+DECLARE_CONFIG_VALUE(MULTI);  // fixme: correct me with more meaningful name, thanks!
+
```

Comment:
as by the default (e.g. when no WORK_MODE is specified) the MULTI is behaving in the legacy mode, seems like we actuall do not need this 'MULTI' mode :), at least I see no check for this value in the rest of the code

Reply:
so looks like we need no special value, just a key to trigger AUTO-like beh for the MULTI

Reply:
OK, then, I will remove the value definition of MULTI_MODE, and  just set the internal key KEY_WORK_MODE to trigger AUTO beh in MULTI, please correct me if I misunderstand.